src/Runtime/jni/src/com/ibm/onnxmlir/OMTensor.java

```diff
@@ -15,25 +15,25 @@
     /* We can use enum but that creates another class
      * which complicates things for JNI.
      */
-    final int ONNX_TYPE_UNDEFINED  = 0;
-    final int ONNX_TYPE_FLOAT      = 1;
-    final int ONNX_TYPE_UINT8      = 2;
-    final int ONNX_TYPE_INT8       = 3;
-    final int ONNX_TYPE_UINT16     = 4;
-    final int ONNX_TYPE_INT16      = 5;
-    final int ONNX_TYPE_INT32      = 6;
-    final int ONNX_TYPE_INT64      = 7;
-    final int ONNX_TYPE_STRING     = 8;
-    final int ONNX_TYPE_BOOL       = 9;
-    final int ONNX_TYPE_FLOAT16    = 10;
-    final int ONNX_TYPE_DOUBLE     = 11;
-    final int ONNX_TYPE_UINT32     = 12;
-    final int ONNX_TYPE_UINT64     = 13;
-    final int ONNX_TYPE_COMPLEX64  = 14;
-    final int ONNX_TYPE_COMPLEX128 = 15;
-    final int ONNX_TYPE_BFLOAT16   = 16;
-
-    final int[] ONNX_TYPE_SIZE = new int[] {
+    final static int ONNX_TYPE_UNDEFINED  = 0;
```

Comment:
Nit: could you add in the comment with which list this "enum" needs to be kept consistent with?